### DISCUSSION TITLE
Prisma - mongodb on production get a error

### DISCUSSION THREAD

**Discussion by hakimLyon** [prisma/prisma#17120] [Q&A] [answered]
```ConnectorError(ConnectorError { user_facing_error: None, kind: AuthenticationFailed { user: "SCRAM failure: Authentication failed." } })```

Pls How to fix that, and deploy on production ?

**Comment by nurul3101** [comment, marked as answer]
Hello @hakimLyon 👋 

Can you try the suggestions mentioned in this StackOverflow [Answer](https://stackoverflow.com/questions/72708560/how-can-i-solve-a-scram-authentication-issue-in-mongodb-with-docker-and-prisma)?

Are you using a MongoDB Database hosted on MongoDB Atlas?

**Reply by hakimLyon** [reply]
Hello @nurul3101, thanks for the response, yes I tried it ,but same error.
 In dev mode ```npm run dev``` it work well. But after build with ```npm run build``` and ```npm run start```  i have this error message ```ConnectorError(ConnectorError { user_facing_error: None, kind: AuthenticationFailed { user: "SCRAM failure: Authentication failed." } })```

I use MongoDB locally to a remote server.  In my .env file i have  value like: 
```DATABASE_URL="mongodb://root:root$@example.com:27033/test?directConnection=true&ssl=true&authSource=admin"```

**Reply by nurul3101** [reply]
Does your local MongoDB server have a replica set configured? It is required to have a replica set configured if you are using a local MongoDB database.
<img width="719" alt="Screenshot 2023-01-05 at 1 37 19 PM" src="https://user-images.githubusercontent.com/13049676/210731662-6c475156-bb08-43ce-b3c7-d97c2de31725.png">

**Reply by hakimLyon** [reply]
Yes it does. 
I found the issue, ```dontenv -e .env``` 🙈.

Now it works  without ```dontenv -e .env``` 😄.
Thanks a lot @nurul3101 💯 

![Capture d’écran du 2023-01-05 12-25-01](https://user-images.githubusercontent.com/64654633/210779813-5f2148fa-63da-4999-99c8-81be09a63c2d.png)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
